pennylane/wires.py

```diff
@@ -123,6 +123,23 @@ def tolist(self):
         """
         return list(self.wire_tuple)
 
+    def at_index(self, idx):
```

Comment:
How does this differ from indexing the wire object directly?

Reply:
Is the difference that if you index a wire object directly, you get a wire back? E.g.,
```python
>>> w = Wires([0, 'q1', 16])
>>> w[1]
Wires(['q1'])
>>> w.at_index(1)
'q1'
```

Reply:
If so, I'm wondering if it makes sense to name this something a bit more descriptive, that indicates you get the 'name' of the wire. Maybe `get_name`?
```python
>>> w = Wires([0, 'q1', 16])
>>> w[1]
Wires(['q1'])
>>> w.get_name(1)
'q1'
```

Reply:
Yes, exactly right. I was thinking of ``get_wire_representation`` but it is very long. ``get_name`` implies that it is a string...

Reply:
I really don't know. I thought ``at_index`` means "at the index i of my Wires object is something called 'q1' ". But if you think ``get_name`` is better we go with that. Or ``get_id``? We could start calling these things wire identifiers...